src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -47,6 +48,22 @@ public function __construct(string $kernelEnvironment, string $projectDirectory)
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('name', InputArgument::OPTIONAL, 'The name of variable to filter for debug.', null, $this->getAvailableVars(...)),
+            ])
+            ->setHelp(<<<'EOT'
+    <info>php %command.full_name% </info>
```

Comment:
To be consistent with other commands, I think we need a sentence that introduces the command. Something like `The <info>%command.name%</info> displays the environment variables configured by dotenv:
`